examples/custom-interaction/javascript-tooltip-and-popover/popover-info.js

```diff
@@ -0,0 +1,219 @@
+const bpmnContainerElt = window.document.getElementById('bpmn-container');
+// TODO enable navigation support (issue with panning)
+const bpmnVisualization = new bpmnvisu.BpmnVisualization(bpmnContainerElt, { mouseNavigationSupport: false });
+
+const diagram = getHorizontalBpmnDiagram();
+bpmnVisualization.load(diagram, { fit: {type: 'Center', margin: 10 } });
+
+
+// user task - task 2
+const task2Elt = bpmnVisualization.htmlElementRegistry.getBpmnHtmlElement('Activity_0kn4d46');
```

Comment:
Maybe to simply and don't have to add a comment, we can rename in the bpmn file the id of the tasks with a explicit id.

Reply:
I will remove the comment, the variable name is clear enough.
About the BPMN element namings, I will probably use the C.1.1 BPMN diagram which has clearer names but in a next PR with other refactoring.